google-chart.html

```diff
@@ -132,11 +132,9 @@
     var pkg = opt_pkg || DEFACTO_CHART_PACKAGE;
     if (pkgPromises[pkg]) return pkgPromises[pkg];
     pkgPromises[pkg] = new Promise(function(resolve) {
-      google.load('visualization', '1', {
-        packages: [pkg],
-        callback: function() {
-          resolve(google.visualization);
-        }
+      google.charts.load('current', {packages: ['corechart']});
```

Comment:
This change would unfortunately reduce the number of chart types available in google-chart, as it's hard-coding the package list. Is there any chance you can use the `pkg` variable above with this loading mechanism?